rero_ils/modules/api.py

```diff
@@ -451,13 +451,26 @@ class IlsRecordsIndexer(RecordIndexer):
 
     record_cls = IlsRecord
 
+    def after_index_record(self, record):
+        """Is called after index and _index_action.
+
+        :param record: indexed record.
+        """
+
```

Comment:
please add `pass` or `NotImplementedError`.

Reply:
I had `pass`, but the linter was saying `unnecessary pass statement`

Reply:
The test: ` Test useless imports` will not succeed with `pass`!